### PR TITLE
Huge cleanup

### PR DESCRIPTION
**Pull request by L3nn0x**

### CODE REVIEW

File: Revise Tests/Revise Tests.csproj [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\NUnit.3.12.0\build\NUnit.props" Condition="Exists('..\packages\NUnit.3.12.0\build\NUnit.props')" />
```
Comment (rminderhoud) [inline Revise Tests/Revise Tests.csproj:3]: Is this default location for nuget packages? Can you add to README how to install it?

File: Revise Tests/Revise Tests.csproj [hunk 2]
```diff
@@ -30,24 +35,21 @@
...
-    <Reference Include="nunit.mocks">
-      <HintPath>..\packages\NUnit.2.5.10.11092\lib\nunit.mocks.dll</HintPath>
-    </Reference>
-    <Reference Include="pnunit.framework">
-      <HintPath>..\packages\NUnit.2.5.10.11092\lib\pnunit.framework.dll</HintPath>
+    <Reference Include="nunit.framework, Version=3.12.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
+      <HintPath>..\packages\NUnit.3.12.0\lib\net45\nunit.framework.dll</HintPath>
     </Reference>
     <Reference Include="SharpDX">
       <HintPath>..\packages\SharpDX.2.0.4\lib\net20\SharpDX.dll</HintPath>
```
Comment (rminderhoud) [inline Revise Tests/Revise Tests.csproj]: Can this reference be removed as well?

File: Revise/AIP/AbilityType.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+﻿#region License
+
+
+
+#endregion
+
+/**
```
Comment (rminderhoud) [inline Revise/AIP/AbilityType.cs:3]: License not inside license region :)

**Review by rminderhoud — COMMENTED** [review]
These two made it into the commit ` .vs/Revise/v16/TestStore/0/004.testlog` and ` .vs/Revise/v16/TestStore/0/testlog./manifest`

I'd say `packages/NUnit.3.12.0` should either be removed with instructions for installing from nuget in readme OR we should add those binaries to LFS with `.gitattributes` file.

**Review by rminderhoud — APPROVED** [review]

**Comment by rminderhoud** [comment]
Let's merge this giganton

**Review by RavenX8 — APPROVED** [review]

**Pull request merged**